plugins/formal-verification/formver.common/src/org/jetbrains/kotlin/formver/PluginConfiguration.kt

```diff
@@ -5,4 +5,14 @@
 
 package org.jetbrains.kotlin.formver
 
-class PluginConfiguration(val logLevel: LogLevel, val behaviour: UnsupportedFeatureBehaviour)
\ No newline at end of file
+class PluginConfiguration(val logLevel: LogLevel, val behaviour: UnsupportedFeatureBehaviour) {
+    private val internalErrorInfos = mutableListOf<String>()
+    val minorErrors = mutableListOf<String>()
```

Comment:
what do you think about making `minorErrors` private and adding two functionns like 
```
    fun addMinorError(error: String){
        minorErrors.add(error)
    }

    fun minorErrorsAction(action: (String) -> Unit) {
        minorErrors.forEach { action(it) }
    }
```
just to have a better encapsulation